Bio/Affy/CelFile.py

```diff
@@ -1,10 +1,10 @@
-# Copyright 2004 by Harry Zuzan.  All rights reserved.
+# Copyright 2004 by Harry Zuzan and Adam Kurkiewicz. All rights reserved.
 # This code is part of the Biopython distribution and governed by its
 # license.  Please see the LICENSE file that should have been included
 # as part of this package.
 
 """
-Classes for accessing the information in Affymetrix cel files.
+Classes for accessing the information in Affymetrix cel files version 3 and 4
 
 Functions:
 read      Read a cel file and store its contents in a Record
```

Comment:
Sorry, I missed a letter out: I meant: "Docstring needs to include the new version 3 and 4 functions for completeness."

i.e. Listing ``read``, ``readV3`` and ``readV4`` or whatever they get called.